src/key.rs

```diff
@@ -992,6 +993,14 @@ impl KeyPair {
     pub fn sign_schnorr(&self, msg: Message) -> schnorr::Signature {
         SECP256K1.sign_schnorr(&msg, self)
     }
+
+    /// This function attempts to erase the contents of the underlying array.
+    /// Note, however, that since this type is `Copy`, ensuring that the contents
+    /// of this array don't end up elsewhere in memory is very subtle. For more
+    /// discussion on this, please see the documentation of the
+    /// [`zeroize`](https://docs.rs/zeroize) crate.
+    #[inline]
+    pub fn non_secure_erase(&mut self) { self.0.non_secure_erase(); }
```

Comment:
```suggestion
    /// Attempts to erase the secret within the underlying array.
    ///
    /// Note, however, that since [`KeyPair`] implements `Copy`, ensuring that the secret data
    /// doesn't end up elsewhere in memory is very subtle. For more discussion on this, please see
    /// the documentation of the [`zeroize`](https://docs.rs/zeroize) crate.
```